src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/initiation/InitiationTaskAttributesMapper.java

```diff
@@ -33,6 +33,36 @@ public InitiationTaskAttributesMapper(ObjectMapper objectMapper) {
 
     public Map<String, Object> mapTaskAttributes(CamundaTask camundaTask, Map<String, CamundaVariable> variables) {
         // Local Variables
+        String type = getType(camundaTask, variables);
+
+        Map<String, Object> attributes = new ConcurrentHashMap<>();
```

Comment:
We are in Spring Singleton (Service annotation) and no need to use ConcurrentHashMap.
new HashMap<>(); is enough, because each thread will have a private stack memory.

Reply:
It's from before, tried to change to HashMap and the tests failed. So, left it back with ConcurrentHashMap.